docs/buildah-bud.md

```diff
@@ -803,22 +803,22 @@ are stored while pulling and pushing images.  Defaults to '/var/tmp'.
 
 ## Files
 
-### `.dockerignore`
+### `.containerignore`/`.dockerignore`
 
-If the file .dockerignore exists in the context directory, `buildah copy` reads
-its contents. Use `--ignorefile` flag to override .dockerignore path location.
-Buildah uses the content to exclude files and directories from the context
-directory, when executing COPY and ADD directives in the
-Containerfile/Dockerfile
+If the .containerignore/.dockerignore file exists in the context directory,
+`buildah copy` reads its contents. Use `--ignorefile` flag to override
+the ignore file path location. Buildah uses the content to exclude files and
+directories from the context directory, when executing COPY and ADD directives
+in the Containerfile/Dockerfile
```

Comment:
What if both files exist in the context directory?  I assume that `.containerignore` will be used and the other is ignored?

Reply:
Correct.